src/test/java/com/github/lombrozo/testnames/rules/RuleTestCaseContainsMockeryTest.java

```diff
@@ -62,4 +66,26 @@ void complaintsAboutMockery() {
             Matchers.containsString(text)
         );
     }
+
+    @ParameterizedTest
+    @MethodSource("classes")
+    void doesNotComplaints(final TestClass test) {
+        final Collection<Complaint> complaints = new RuleTestCaseContainsMockery(
+            new ListOf<>(test.all()).get(0), 2
```

Comment:
@h1alexbel it seems you are testing only the first method here? Please, correct me if I'm wrong.

Reply:
@volodya-lombrozo yes, good catch, thanks